data/human/news.txt

```diff
@@ -1227,3 +1336,31 @@ news "graffiti on spaceport wall"
 			"You think piracy is bad and want to stop us? You and what army?"
 			"Messing with the Syndicate, Does That Sound Like Fun to You?"
 			"83 percent of all statistics are just made up."
+			"Ask not what your planet can do for you; ask what you can do for your planet."
+			"Fair is foul, and foul is fair - Shakespeare, Macbeth"
+			"For want of a shoe, the crewmember was lost. / For want of a crewmember, the ship was lost. / For want of a ship, the fleet was lost. / For want of a fleet, the battle was lost. And all for the want of a shoe."
+			"Shoot for the stars, but make sure you miss or your hull might not last."
+			"Clippy the paperclip was here."
+			"Kilroy was here."
+			"Lassy ran by."
+			"Mr Pappa was here."
+			"The Easter Bunny was here."
+			"The Joker was here."
+			"Veronica Oxygenpocket was here."
+			"Vincent van Gogh was here."
+			"Beethoven and Mozart made music here."
+			"Foo went to a Bar."
+			"Hansel and Gretel experienced fairy tales here."
+			"Jane and John Doe were here."
+			"Jack and Jill went up a hill here."
+			"Johnny Walker and Jack Daniels had a drink here."
+			"The Pink Elephant was in a room around here."
+			"The Three Stooges were here."
```

Comment:
I'm not really sure about these kinds of lines. They seem kind of random and out of place.

Reply:
Specifically lines 1350 to 1358? Or also some of the other lines here?

Reply:
A lot of the lines in this part are kind of strange to me.

Reply:
I removed a number of those lines. Please let me know if any of the remaining lines are still out of place.